core/graphelier-service/api/hndlrs/orderhandler_test.go

```diff
@@ -0,0 +1,69 @@
+package hndlrs_test
+
+import (
+	"graphelier/core/graphelier-service/models"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+
+	"graphelier/core/graphelier-service/api/hndlrs"
+	. "graphelier/core/graphelier-service/utils/test_utils"
+)
+
+func TestFetchOrderInfoSuccess(t *testing.T) {
+
+	mockedDB := MockDb(t)
+	defer Ctrl.Finish()
+	messages := []*models.Message{&models.Message{Direction: -1, Instrument: "test", Type: models.NewOrder, OrderID: 1, Price: 100, ShareQuantity: 10, Timestamp: 1, SodOffset: 1}}
+	mockedDB.EXPECT().
+		GetSingleOrderMessages("test", int64(1579564800000000000), int64(1579651200000000000), int64(1)).
+		Return(messages, nil)
+
+	var orderInfo models.OrderInfo
+	err := MakeRequest(
+		hndlrs.FetchOrderInfo, // Function under test
+		mockedDB,
+		"GET",
+		"/order/test/1/2",
+		map[string]string{
+			"instrument": "test",
+			"orderID":    "1",
+			"timestamp":  "1579580821000000000",
+		},
+		&orderInfo,
+	)
+	assert.Nil(t, err)
+	assert.EqualValues(t, 100, orderInfo.Price)
```

Comment:
Can you add a comment describing what you are asserting for here? To gain a higher level perspective than "price should be 100"

Reply:
This test is mostly used to see if the the db gets called with the correct parameters these assertions are just there to confirm that the handler doesn't change anything

Reply:
Can you add a comment saying that in the file? To keep in line with #150